src/language/providers/intellisense/elementItems.ts

```diff
@@ -49,17 +49,17 @@ export const elementCompletion = (definedVariables, nsPrefix) => {
       },
       {
         item: 'dfdl:assert',
-        snippetString: '<' + nsPrefix + 'assert $0',
+        snippetString: '<' + 'dfdl:assert $0',
```

Comment:
Change these from something like this:
 `snippetString: '<' + 'dfdl:assert $0',`
 To something like this:
 `snippetString: '<dfdl:assert $0',`